### DISCUSSION TITLE
Teslamate displayed nothing when logged in localhost:4000

### DISCUSSION THREAD

**Discussion by lihongzheng-nlp** [teslamate-org/teslamate#4271] [Q&A]
I installed teslamate on my M1 iMac with docker desktop following the official installation instruction, I logged in the localhost:4000 webpage with tokens successfully, but there is nothing on the page, totally empty. When logged in the localhost:3000, there is nothing either.
<img width="1134" alt="teslamate" src="https://github.com/user-attachments/assets/ba2b3775-34f1-4730-88fb-d5acf1b346ef">
So, how to solve the problem?

**Comment by cwanja** [comment]
What's in the logs?

**Reply by lihongzheng-nlp** [reply]
[database.log](https://github.com/user-attachments/files/17427315/database.log)
[teslamate-teslamate-1.log](https://github.com/user-attachments/files/17427316/teslamate-teslamate-1.log)
here are the database log and teslamate-1 log yesterday night, I'm not quite familiar with docker, so I'm not sure how to check it, but ChatGPT told me to check logs.

**Comment by ktsenvolt** [comment]
Have you woken up the car?
